Not compatible with latest Highcharts 11.4.6

### Describe the bug
We updated from Highcharts 11.4.3 to 11.4.6 and now get compile errors.

```
      Error: apps/deployment/src/app/deployments/deployments.component.html:29:12 - error TS2322: Type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").Options' is not assignable to type 'Highcharts.Options'.
        Types of property 'accessibility' are incompatible.
          Type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").AccessibilityOptions | undefined' is not assignable to type 'Highcharts.AccessibilityOptions | undefined'.
            Type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").AccessibilityOptions' is not assignable to type 'Highcharts.AccessibilityOptions'.
              Types of property 'announceNewData' are incompatible.
                Type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").AccessibilityAnnounceNewDataOptionsObject | undefined' is not assignable to type 'Highcharts.AccessibilityAnnounceNewDataOptionsObject | undefined'.
                  Type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").AccessibilityAnnounceNewDataOptionsObject' is not assignable to type 'Highcharts.AccessibilityAnnounceNewDataOptionsObject'.
                    Types of property 'announcementFormatter' are incompatible.
                      Type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").AccessibilityAnnouncementFormatter | undefined' is not assignable to type 'Highcharts.AccessibilityAnnouncementFormatter | undefined'.
                        Type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").AccessibilityAnnouncementFormatter' is not assignable to type 'Highcharts.AccessibilityAnnouncementFormatter'.
                          Types of parameters 'updatedSeries' and 'updatedSeries' are incompatible.
                            Type 'Highcharts.Series[]' is not assignable to type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").Series[]'.
                              Type 'Highcharts.Series' is not assignable to type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").Series'.
                                Types of property 'area' are incompatible.
                                  Type 'Highcharts.SVGElement | undefined' is not assignable to type 'import("D:/Git/cloud-ui/node_modules/highcharts/highcharts.src").SVGElement | undefined'.
                                    Type 'SVGElement' is missing the following properties from type 'SVGElement': setPolygon, setTextPath

      29           [options]="chartDeployments"
                    ~~~~~~~

        apps/deployment/src/app/deployments/deployments.component.ts:29:16
          29   templateUrl: './deployments.component.html',
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          Error occurs in the template of component DeploymentsComponent.
 ```


### Expected behavior
Expect code to build

### Demo
This is our html code:

```
        <highcharts-chart
          [Highcharts]="Highcharts"
          [options]="chartDeployments"
          [update]="updateMap.deployments">
        </highcharts-chart>
```

#### Steps required to recreate the problem in the demo:
1. ...

### Setup used
 - NodeJS version ... 20.13.0
 - Angular version ... 17.3.8
 - TypeScript version ... 5.4.5
etc.


<!-- ASANA TASK ID (do not delete): 1207772630017710 -->

--- a/highcharts-angular/src/lib/highcharts-chart.component.ts
+++ b/highcharts-angular/src/lib/highcharts-chart.component.ts
@@ -13,7 +13,7 @@ export class HighchartsChartComponent implements OnDestroy, OnChanges {
   @Input() callbackFunction: Highcharts.ChartCallbackFunction;
   @Input() oneToOne: boolean; // #20
   @Input() runOutsideAngular: boolean; // #75
-  @Input() options: Highcharts.Options;
+  @Input() options: Highcharts.Options | HighchartsESM.Options;
   @Input() update: boolean;
 
   @Output() updateChange = new EventEmitter<boolean>(true);

